crates/matrix-sdk-crypto/CHANGELOG.md

```diff
@@ -69,7 +69,7 @@ Breaking changes:
   previously-verified, unverified and verified.
   ([#3877](https://github.com/matrix-org/matrix-rust-sdk/pull/3877))
 
-- Where `EncryptionInfo` is returned it may include the new `PreviouslyVerified`
+- Where `EncryptionInfo` is returned it may include the new `VerificationViolation`
```

Comment:
driveby comment: given we've already released this stuff, can you put a separate changelog entry higher up, please?

Reply:
Thanks, I dithered about this and chose the wrong choice!

Reply:
Fixed in a3da650b06